analyzers/src/SonarAnalyzer.Rules.SourceGenerator/packages.lock.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
This file is a mess and shouldn't be here (wrong directory)

Reply:
I'll delete it. I was just highlighting that fact on self-review so we don't forget about it :)